cmd/goQuery/cmd/root.go

```diff
@@ -90,6 +90,8 @@ func init() {
 	flags.BoolVarP(&cmdLineParams.Sum, "sum", "", false, helpMap["Sum"])
 	flags.BoolVarP(&cmdLineParams.Version, "version", "v", false, "Print version information and exit\n")
 
+	flags.StringVar(&cmdLineParams.ProfilingOutputDir, "profiling-output-dir", "", "Enable and set directory to store CPU and memory profiles")
```

Comment:
I wouldn't make this a property of the query, but rather list this in pkg/conf.

E.g.: const ProfilingOutputDir = profilingKey + ".output-dir"

Reply:
Sure think. Like this: 235c67c2b166ee0a085616ee6fa17589cd94b2a5 ?

Reply:
Almost 😄 . Let me make a suggestion before we ping-pong.

Reply:
@fako1024 : please see latest push regarding the re-arrangement of the function calls.